install.sh

```diff
@@ -5,13 +5,23 @@
 
 set -e
 
+for arg in "$@"; do
+    shift
+    case "$arg" in
+        "--root") set -- "$@" "-r" ;;
+        *) set -- "$@" "$arg"
+    esac
+done
+
 while getopts ":r" arg; do
     case "${arg}" in
         "r") override_root=1 ;;
```

Comment:
Couldn't we just change this to `root`?

Reply:
well, most of scripts/cli provide short version an long version so that's why it has both 🤷‍♀️

Reply:
Yeah, but that doesn't always happens as it is a rule that doesn't apply to all flags. In this case, we are talking about a flag that shouldn't be used outside of exceptions, so I would say that we shouldn't have a short version for this one, and just simplify and only accept `--root`

Reply:
alright